src/jsc/jsc_class.hpp

```diff
@@ -276,6 +278,10 @@ inline JSValueRef ObjectWrap<ClassType>::get_property(JSContextRef ctx, JSObject
             // Out-of-bounds index getters should just return undefined in JS.
             return Value::from_undefined(ctx);
         }
+        catch (Results::OutOfBoundsIndexException &) {
+            // Out-of-bounds index getters should just return undefined in JS.
+            return Value::from_undefined(ctx);
+        }
```

Comment:
This isn't necessary since `OutOfBoundsIndexException` inherits from `out_of_range`. I removed it locally and all tests still pass. I can commit that change if you want.